### PR TITLE
Feature/extract system bots

### PR DESCRIPTION
**Pull request by marioosh**
Don't merge, It will be merged by gitflow! For Review Only.

### CODE REVIEW

File: src/main/scala/io/scalac/slack/SlackBot.scala [hunk 1]
```diff
@@ -2,25 +2,24 @@ package io.scalac.slack
...
-import io.scalac.slack.api.{BotInfo, Start}
-import io.scalac.slack.common.DefaultBotBundle
-import io.scalac.slack.websockets.{WSActor, WebSocket}
+import io.scalac.slack.api.Start
+import io.scalac.slack.common.actors.SlackBotActor
+import io.scalac.slack.common.{DefaultBotBundle, Shutdownable, UsersStorage}
+import io.scalac.slack.websockets.WebSocket
 
-object SlackBot {
+object SlackBot extends Shutdownable {
```
Comment (pjazdzewski1990) [inline src/main/scala/io/scalac/slack/SlackBot.scala]: What is`Shutdownable`? Looks like it's part of this project, but I can't see it

**Comment by pjazdzewski1990** [comment]
Please make sure that `Shutdownable` is part of the repo. Besides this :+1:

File: src/main/scala/io/scalac/slack/SlackBot.scala (hunk 1 above)
Comment (marioosh) [inline src/main/scala/io/scalac/slack/SlackBot.scala]: it's part of `core` but don't forget about looking into `develop` branch if you're looking for most fresh changes. :)

**Comment by marioosh** [comment]
merged to `develop` instead of  `master`

**Pull request closed without merging**